### PR TITLE
Add reference to TEASER repository, #700

### PR DESCRIPTION
**Pull request by PRemmen**
This adds a reference to the TEASER repository for parameter calculation, solves #700

### CODE REVIEW

File: IBPSA/ThermalZones/ReducedOrder/package.mo [hunk 1]
```diff
@@ -29,6 +29,11 @@ package ReducedOrder
     smooth=Smooth.None)}), Documentation(info="<html>
     <p>This package contains models for reduced building physics of thermal
     zones and accompanying models for consideration of solar radiation and
-    radiation transmission through windows. </p>
+    radiation transmission through windows.</p>
+    <p>The python package TEASER <a
+    href=\"https://github.com/RWTH-EBC/TEASER\">
+    https://github.com/RWTH-EBC/TEASER</a> can be used to calculate reduced
+    order parameters and generate ready-to-tun models.
```
Comment (mlauster) [inline IBPSA/ThermalZones/ReducedOrder/package.mo]: Typo, "ready-to-run"

**Comment by mlauster** [comment]
We should add this sentence to `RC.UsersGuide` as well, I think this is the major source of information and most users will have a look here (if they read the docs at all).

**Comment by mlauster** [comment]
@mwetter, all tests are failing, probably because it's issued from outside ibpsa/modelica ,@PRemmen is not yet part of it. As this PR only includes changes on documentation, I vote for merging it. Objections from your side?

**Comment by thorade** [comment]
Peter created the PR against master. In the top right corner there is a button `Edit`. You can change the branches used for the Pull Request, change it so that the PR is not against master, but against the issue branch here on the repository (it has the same name). You can safely merge Peters code to the issue branch and then create a new Pull Request, asking to merge the IBPSA/issue-branch into IBPSA/master. That PR should run without problems.
Or, add Peter to the IBPSA organization.

**Comment by PRemmen** [comment]
@thorade thanks for thehint. I think it was supposed to be like this any way :+1: 

@mlauster I changed base so you can hopefully savely merge into `issue700_romDocumentation` and then create a PR for master by yourself

**Pull request merged**